Jenkinsfile

```diff
@@ -35,6 +38,7 @@ pipeline {
             GOOGLE_SERVICES = credentials('google-services')
           }
           steps {
+            setNexusNpm()
```

Comment:
Why having the setup nexus in the pipeline and in the Dockerfile?

Reply:
In Dockerfile there is step to install Cordova as root, that is why I added that to Dockerfile. Then it switches to `jenkins` user, I was not sure if it will use root npm config, so added it in the steps as well. If it use the same `.npmrc`, then it can be removed.

Reply:
Maybe there is a way to set the npm registry globally for all users so that we can set it only in the container

Reply:
I think that we would still need to set it in Dockerfile, because there is a step in it to install Cordova. But in the dockerfile it should be possible to set things globally with `-g` flag. Then we would not need to set it again in container theoretically. wdyt?